src/main/java/it/eng/idsa/businesslogic/service/BrokerService.java

```diff
@@ -0,0 +1,9 @@
+package it.eng.idsa.businesslogic.service;
+
+import de.fraunhofer.iais.eis.Message;
+
+public interface BrokerService {
+	
+	public void sendBrokerRequest(Message message, String selfDescription);
```

Comment:
Second parameter is not always selfDescription, maybe to rename it to payload?
When sending query request, payload is not selfDescription

Reply:
Sure, no problem